source/geh_calculated_measurements/tests/electrical_heating/electrical_heating_tests/infrastructure/measurements/conftest.py

```diff
@@ -0,0 +1,56 @@
+import datetime
+from decimal import Decimal
+
+import pytest
+from geh_common.testing.delta_lake.delta_lake_operations import create_database, create_table
+from pyspark.sql import SparkSession
+
+from geh_calculated_measurements.electrical_heating.infrastructure import (
+    MeasurementsGoldDatabaseDefinition,
+    MeasurementsGoldRepository,
+    electrical_heating_v1,
+)
+
+
+@pytest.fixture(autouse=True)
+def test_table_inserted(spark: SparkSession) -> None:
```

Comment:
Naming: It's not a test. Suggestion:
```suggestion
def measurments_gold_with_data(spark: SparkSession) -> None:
```